ui-guidelines/css/resources.css

```diff
@@ -0,0 +1,128 @@
+/*==== TYPOGRAPHY ====*/
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+
+.font-weights {
+	width: 100%;
+	height: 10em;
+}
+.weight-sample {
+	width: 20%;
+	float: left;
+	margin-right: 5%;
+	border: 1px solid #fce3da;
+	border-radius: 8px;
+}
+.big-letter{
+	font-size: 6em;
+	font-family: 'Open Sans', Arial, sans-serif;
+}
+.light {font-weight: 300;}
+.normal {font-weight: 500;}
+.medium {font-weight: 600;}
+.bold {font-weight: 700;}
+.text-size {margin-left: 1.5em;}
+
+.typography, .typography p {
+	font-family: 'Open Sans', Arial, sans-serif;
+}
+
+.text-h1 {font-size: 2.5em;}
+.text-h2 {font-size: 1.5em;}
+.text-h3 {font-size: 1.25em;}
+
+.text-x-huge {font-size: 2.5em;}
+.text-huge {font-size: 1.875em;}
+.text-xx-large {font-size: 1.4em;}
+.text-x-large {font-size: 1.133em;}
+.text-large {font-size: 1.1em;}
+.text-xx-medium {font-size: 1.066em;}
+.text-x-medium {font-size: 1em;}
+.text-medium {font-size: .933em;}
+.text-small {font-size: .9em;}
+.text-x-small {font-size: .866em;}
+.text-xx-small {font-size: .85em;}
+.text-minimum {font-size: .8em;}
+.text-x-minimum {font-size: .75em;}
+.text-xx-minimum {font-size: .625em;}
+
+/*==== COLORS ====*/
+hr {
+	margin-top: 0px;
+	border-top: 1px solid #333;
+}
+.color-row {
+	display: inline-block;
+}
+.color-group {
+	display: inline-block;
+	float: left;
+	margin: 1em 3em 1.5em 0;
+}
+.color-group-bottom { 
+	margin-bottom: 0em;
+}
+.color-box {
+	display: block;
+	float: left;
+	height: 5em;
+	width: 7em;
+	border: 1px solid transparent;
+	border-radius: 4px;
+	margin-bottom: 1em;
+}
+.color-label {
+	font-size: .9em;
+	font-family: 'Quantico', sans-serif;
+}
+/*Primary colors*/
+.clr-primary {background-color: #FF7543;}
+.clr-primary-hover {background-color: #F15A10;}
+.clr-secondary {background-color: #2b84b1;}
+.clr-secondary-hover {background-color: #006094;}
+.clr-neutral {background-color: #9b9b9b;}
+.clr-neutral-hover {background-color: #667076;}
+.clr-negative {background-color: #E85365;}
+.clr-negative-hover {background-color: #D92D42;}
+/*Secondary colors*/
+.clr-action-bar {background-color: #106D95;}
+.clr-action-bar-line {background-color: #7AAABE;}
+.clr-general-bar {background-color: #667076;}
+.clr-general-bar-line {background-color: #BBB;}
+.clr-action-bar-preview {background-color: #185068;}
+.clr-form-border {background-color: #b1d2e0;}
+.clr-form-data {background-color: #3B6D86;}
+.clr-form-dropdown {background-color: #747474;}
+.clr-focus-background {background-color: #FCB700;}
+/*Text colors*/
+.clr-text {background-color: #333;}
+.clr-text-white {background-color: #FFF; border: 1px solid #333;}
```

Comment:
I would expect CSS styles for letter `color`, not the `background` and `border` properties.